### ISSUE TITLE
[WC 3.9-rc3] - Address Accessibility issue with Filter by Price block.

### ISSUE THREAD

**Issue by nerrad** [woocommerce/woocommerce-gutenberg-products-block#1592]
This was reported from internal testing of WC 3.9-rc3:

> When you change the minimum price of the filter and then press tab (expecting to change the max price in the filter), the filter and all products block starts refreshing.. and after the refresh, the focus s not on the maximum price field, so the user would need to cycle through all other focus fields on the page and then set the max price filter, have demonstrated the same in the GIF below (wait till the end as I cycle through all focus fields and then arrive at max price filter field 🙂

GIF: https://d.pr/i/6V7p7M

I have not confirmed this yet.

**Comment by Aljullu** [comment]
I'm unable to reproduce this one. We have this check to avoid triggering `onChange` when moving focus from one input field to the other:

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/assets/js/base/components/price-slider/index.js#L188-L197

I asked some more info regarding browser and whether we can get a link to that page to test it ourselves.

**Comment by Aljullu** [comment]
My guess is this is a conflict with another plugin (probably [Brezee](https://wordpress.org/plugins/breeze/)?), I was trying to reproduce it locally with no luck yet.

I asked to try disabling Brezee and see if that helps (pb22l9-5T-p2#comment-184).

I'm will unassign myself from this issue until we get more details on this.

**Comment by nerrad** [comment]
This was reported as being a conflict with other plugins so I'm going to go ahead and close this.
